tasks.py

```diff
@@ -403,9 +409,10 @@ def check_migrations(context):
         "label": "specify a directory or module to test instead of running all Nautobot tests",
         "failfast": "fail as soon as a single test fails don't run the entire test suite",
         "buffer": "Discard output from passing tests",
+        "verbose": "Toggle verbose test output",
```

Comment:
While you're touching this one (and adding the `integration_test` task below), can we become consistent on capitalization ("Discard", "Toggle") or not ("save", "specify", "fail") of the help strings?

Reply:
Yes thanks.